ethereum-common/src/events.rs

```diff
@@ -52,22 +52,23 @@ fn index_events(events: Events) -> Result<Keys, Error> {
 
 #[substreams::handlers::map]
 fn filtered_events(query: String, events: Events) -> Result<Events, Error> {
-    let filtered: Vec<Event> = events
-        .events
-        .into_iter()
-        .filter(|e| {
-            if let Some(log) = &e.log {
-                evt_matches(log, &query).expect("matching calls from query")
-            } else {
-                false
-            }
-        })
-        .collect();
+    return _filtered_events(query, events);
```

Comment:
```suggestion
   _filtered_events(query, events)
```

Like this, it removes warning from Rust when compiling.